### PR TITLE
RHDEVDOCS-6277 - Updating GitOps docs to improve content for argo CD resource, default…

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
Jira issue: [RHDEVDOCS-6277](https://issues.redhat.com/browse/RHDEVDOCS-6277) 

Aligned team: DevTools 
Cherry-pick to versions: `gitops-docs-1.13` and later

Docs preview:

- [Glossary of common terms for OpenShift GitOps](https://85542--ocpdocs-pr.netlify.app/openshift-gitops/latest/understanding_openshift_gitops/about-redhat-openshift-gitops.html#gitops-openshift-go-common-terms_about-redhat-openshift-gitops)
- [Setting up an Argo CD instance](https://85542--ocpdocs-pr.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance.html)

SME review: Completed by @anandf

QE review: @varshab1210

Peer review:  

Additional information: N/A

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@anandf Please review this PR and provide feedback. Thanks!

File: argocd_instance/setting-up-argocd-instance.adoc [hunk 1]
```diff
@@ -8,17 +8,26 @@ toc::[]
 
 By default, {gitops-title} installs an instance of Argo CD in the `openshift-gitops` namespace with additional permissions for managing certain cluster-scoped resources. This default Argo CD instance is also called as the default cluster-scoped instance.
 
+To prevent the default Argo CD instance from starting in the `openshift-gitops` namespace, you can use the `openshift-gitops-operator` subscription and configure the `DISABLE_DEFAULT_ARGOCD_INSTANCE` environment variable in it by setting the value to `true`.
+
```
Comment (anandf) [inline argocd_instance/setting-up-argocd-instance.adoc:12]: Wondering if we should mention it as `setting the string value to `"true"``

File: argocd_instance/setting-up-argocd-instance.adoc [hunk 2]
```diff
@@ -8,17 +8,26 @@ toc::[]
...
+* Restricted pod security admission (PSA) labels are applied to the `openshift-gitops` namespace to ensure compliance with {OCP} standards. If you are running additional workloads in this namespace, such as monitoring or logging, ensure that they comply with the restricted PSA requirements. If compliance is not feasible, consider using a user-defined, cluster-scoped Argo CD instance, where PSA labels are not applied or controlled by the {gitops-shortname} Operator.
 ====
 
 To manage cluster configurations or deploy applications, you can install and deploy a new user-defined Argo CD instance. By default, any new user-defined instance has permissions to manage resources only in the namespace where it is deployed.
 
+You can create a user-defined Argo CD instance in any namespace, other than the `openshift-gitops` namespace.
+
+[IMPORTANT]
+====
+If you want to create a user-defined Argo CD instance within the `openshift-gitops` namespace, set the `DISABLE_DEFAULT_ARGOCD_INSTANCE` flag value in the `openshift-gitops-operator` subscription to `true` and do not name the instance as `openshift-gitops`.
```
Comment (anandf) [inline argocd_instance/setting-up-argocd-instance.adoc:28]: Wondering if we should mention it  `"true"`

File: argocd_instance/setting-up-argocd-instance.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply argocd_instance/setting-up-argocd-instance.adoc:12]: Addressing it right away.

**Comment by Srivaralakshmi** [comment]
@anandf I have addressed your feedback. PTAL and let me know if everything checks okay. Thanks!

**Review by anandf — APPROVED** [review]
LGTM

**Comment by Srivaralakshmi** [comment]
@varshab1210 Please review this PR and provide feedback. Thanks!

**Review by xenolinux — COMMENTED** [review]
LGTM

**Pull request merged**